packages/usa-modal/src/index.js

```diff
@@ -284,6 +277,18 @@ modal = {
       const modalId = modalWindow.id;
       setUpModal(modalWindow);
 
+      // Set the value for temporary body padding
+      // that will be applied when the modal is open.
+      // Value is created by checking for initial body padding
+      // and adding the width of the scrollbar.
+      const initialBodyPadding = window
+        .getComputedStyle(document.body)
+        .getPropertyValue("padding-right");
+      const temporaryBodyPadding = `${
+        parseInt(initialBodyPadding.replace(/px/, ""), 10) +
+        parseInt(SCROLLBAR_WIDTH.replace(/px/, ""), 10)
+      }px`;
+
```

Comment:
I'm curious... have we tried removing this entirely?

The padding on my body didn't match up with your values in the testing. On storybook it was 16px instead of 32px and sandbox stayed at 38px.

I noticed that there weren't any visible changes so I did some light testing on removing the initial padding check and update. I didn't notice any content shift or any unintended behaviors!

I was curious if you saw something different than I do or if you've tried removing this portion of the javascript in general 🤔

Reply:
@mahoneycm Could you try logging the values for `SCROLLBAR_WIDTH`, `temporaryBodyPadding`, and `initialBodyPadding` after the modal initializes and let me know what you see? I am curious if it is registering `SCROLLBAR_WIDTH` as 0.

Reply:
You've got good intuition!

<img width="263" alt="Screenshot 2023-09-14 at 2 02 44 PM" src="https://github.com/uswds/uswds/assets/25211650/79d88576-2601-4762-a4c6-5750ca925c64">